surveys/2024-annual-survey/translations/de.md

```diff
@@ -333,20 +333,20 @@ Type: matrix
 Rows:
 
 - Logik für Tupel variabler Länge implementieren
-- Quelltext über mehrere crates verteilen (Orphan Rule)
+- Quelltext über mehrere crates verteilen (z.B. Orphan Rule)
 - Iteratoren manuell implementieren
-- Nicht genug in konstanten funktionen tun können
+- Nicht genug in konstanten Funktionen tun können
 - Dynamische Bibliotheken/Plugins implementieren
 - Mit anderen Sprachen zusammenarbeiten (z.B. C oder C++)
 - Strukturierte Nebenläufigkeit mit Async Code erreichen
 - Async-Laufzeit-agnostischen Code schreiben
 - Korrekten unsafe Code schreiben
 - Der Borrow Checker lehnt meinen validen Code ab
-- Schlechte Performance zur Programmlaufzeit
+- Langsame Performance zur Programmlaufzeit
```

Comment:
German "schlecht" just means "bad", so it could be misunderstood as high memory usage. The English version says "slow" (German "langsam").